In [1]:
import numpy as np
import matplotlib.pyplot as plt
from scipy import integrate
from scipy.special import gamma as gamma 

In [2]:
def tdist(t,nu):
    return  gamma((nu+1)*0.5 )/(gamma(nu*0.5)*np.sqrt(np.pi*nu)) * (1.0 +t**2/nu) **(-(nu+1.0)*0.5)

In [3]:
nu=14
t_arr = np.linspace(-10,10,1000)

ss = 0
for tval in t_arr:
    prob = integrate.quad(tdist, -np.inf, tval, args=(nu))
    if prob[0] >= 0.975:
        print(t_arr[ss-1])
        break
    ss+=1

2.1321321321321314


Entonces,

$$
t_{(0.025)} = 2.1321
$$

El intervalo de confianza obtenido es:

$$
IC = [0.06,\ 4.094]
$$